services/director-v2/src/simcore_service_director_v2/api/routes/dynamic_services.py

```diff
@@ -333,6 +333,7 @@ async def update_projects_networks(
         RabbitMQClient, Depends(get_rabbitmq_client_from_request)
     ],
 ) -> None:
+    # NOTE: This needs to be called to update networks only when adding, removing, or renaming a node.
```

Comment:
I would put this as a comment inside the function not here. Maybe as a doctoring